aten/src/ATen/native/native_functions.yaml

```diff
@@ -601,6 +601,9 @@
 - func: affine_grid_generator_backward(Tensor grad, int[] size, bool align_corners) -> Tensor
   variants: function
 
+- func: _is_true(Tensor self) -> bool
+  variants: function, method
```

Comment:
for this to fully work with nested functorch transforms it likely needs to be marked as CompositeExplicitAutograd and non_differentiable.

Concretely the problematic case is:

```
def f(x):
  assert torch.ops.aten._is_true(x > 0)
  return x
  
x = torch.tensor([1., 2., 3.])
vmap(grad(f))(x)
```

If `_is_true` is composite, then it'll decompose into the .data_ptr() call before it gets to vmap, and vmap will yell about the data_ptr call.